src/components/Navbar/SettingsDrawer/SearchSelectionBody.module.scss

```diff
@@ -25,6 +25,19 @@
   margin-block-end: var(--spacing-small);
 }
 
+.searchInput {
```

Comment:
This is a duplicate of `.searchInput` css inside `src/components/Navbar/SearchDrawer/Header/Header.module.scss`. While this is not a major issue but I think a much better approach is to create a new component called `SearchInput` for example that can be reused by both the search drawer and translations search so we will be sure they are 100% consistent and easier to update in the future.